### PR TITLE
[bitnami/postgresql] Add nodeSelector to Backup Cronjob spec

### PR DESCRIPTION
**Pull request by davek804**
### Description of the change

Straightforward change to allow the backup CronJob within Postgresql to consume a nodeSelector.

### Benefits

This allows teams that use many node labels to customize where their backup cronjob pods reside/execute from. For my work, this is rather crucial as I have networking segments that don't allow access to databases such as Postgres. Prior to this PR, it's possible for the CronJob to spin up in a location where it does not have access to the Postgres DB. 

### Possible drawbacks

N/A. This PR includes no end-product changes unless the new attribute is defined by the user.

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/postgresql/Chart.yaml [hunk 1]
```diff
@@ -35,4 +35,4 @@ maintainers:
 name: postgresql
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/postgresql
-version: 12.11.3
+version: 12.11.4
```
Comment (Mauraza) [inline bitnami/postgresql/Chart.yaml:38]:
```suggestion
version: 12.12.3
```

**Review by Mauraza — COMMENTED** [review]
Hi @davek804,

Thanks for the contribution! It should be a minor version because it's an improvement.

**Comment by davek804** [comment]
@Mauraza it looks like the CI/CD pipeline for this PR failed. I believe there's no input I can provide to assist, but if there are any code changes I can help facilitate I am available.

**Comment by Mauraza** [comment]
Hi @davek804,

Now the last version is 12.12.4 so your change is already released. 
```bash
$ helm search repo postgresql
NAME                 	CHART VERSION	APP VERSION	DESCRIPTION                                       
bitnami/postgresql   	12.12.4      	15.4.0     	PostgreSQL (Postgres) is an open source object-...
```

**Pull request merged**